NixOS: Can't build the plugin

on hyprland 0.45.0

```
error: builder for '/nix/store/1z7alrk3zbgr17a6msdpjiflgwvh1zwz-split-monito
r-workspaces-0.1.drv' failed with exit code 1;
       last 25 log lines:
       >     bindir       : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/bin
       >     buildtype    : plain
       >     includedir   : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/include
       >     infodir      : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/share/info
       >     libdir       : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/lib
       >     libexecdir   : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/libexec
       >     localedir    : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/share/locale
       >     mandir       : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/share/man
       >     prefix       : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1
       >     sbindir      : /nix/store/pxyiy9zkkkspc2pagwdhphhi17jbnckk-spli
t-monitor-workspaces-0.1/sbin
       >     wrap_mode    : nodownload
       >
       > Found ninja-1.12.1 at /nix/store/nwsn9hg84pf9rpy3bgdpssa2b6qc7w3p-n
inja-1.12.1/bin/ninja
       > mesonConfigurePhase: enabled\ parallel\ building
       > Running phase: buildPhase
       > build flags: -j3
       > [1/2] Compiling C++ object libsplit-monitor-workspaces.so.p/src_mai
n.cpp.o
       > FAILED: libsplit-monitor-workspaces.so.p/src_main.cpp.o
       > g++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I../include -I/nix
/store/4m3nzafgkkpfx89f2wvqacd86qhzdxfi-hyprland-0.45.0+date=2024-11-22_745a
82c-dev/include -I/nix/store/4m3nzafgkkpfx89f2wvqacd86qhzdxfi-hyprland-0.45.
0+date=2024-11-22_745a82c-dev/include/hyprland/protocols -I/nix/store/4m3nza
fgkkpfx89f2wvqacd86qhzdxfi-hyprland-0.45.0+date=2024-11-22_745a82c-dev/inclu
de/hyprland -I/nix/store/f40qqcxl63lgfygc22x1h9n9vyd76zcz-pixman-0.43.4/incl
ude/pixman-1 -I/nix/store/kypqq0shrqvmjpk293cmk4jzw9g20nwq-libdrm-2.4.123-de
v/include -I/nix/store/kypqq0shrqvmjpk293cmk4jzw9g20nwq-libdrm-2.4.123-dev/i
nclude/libdrm -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFF
SET_BITS=64 -Wall -Winvalid-pch -std=c++23 -DWLR_USE_UNSTABLE -fno-gnu-uniqu
e -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_main.cpp.o -MF libsplit
-monitor-workspaces.so.p/src_main.cpp.o.d -o libsplit-monitor-workspaces.so.
p/src_main.cpp.o -c ../src/main.cpp
       > ../src/main.cpp: In function 'const std::string& getWorkspaceFromMo
nitor(const PHLMONITOR&, const std::string&)':
       > ../src/main.cpp:74:59: error: 'class CCompositor' has no member nam
ed 'getWindowsOnWorkspace'; did you mean 'getNextWindowOnWorkspace'?
       >    74 |             if (workspacePtr == nullptr || g_pCompositor->g
etWindowsOnWorkspace(workspacePtr->m_iID) == 0) {
       >       |                                                           ^
~~~~~~~~~~~~~~~~~~~~
       >       |                                                           g
etNextWindowOnWorkspace
       > ninja: build stopped: subcommand failed.
```

my chungus config files:
- [configuration.nix](https://github.com/hedonicadapter/dotfiles/blob/main/main/nixos/configuration.nix)
- [home.nix](https://github.com/hedonicadapter/dotfiles/blob/main/main/home-manager/home.nix)
- [flake.nix](https://github.com/hedonicadapter/dotfiles/blob/main/main/flake.nix)

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -71,7 +71,7 @@ const std::string& getWorkspaceFromMonitor(const PHLMONITOR& monitor, const std:
         for (const auto& workspaceName : g_vMonitorWorkspaceMap[monitor->ID]) {
             PHLWORKSPACE workspacePtr = g_pCompositor->getWorkspaceByName(workspaceName);
             // the workspace we want is either not yet created (=nullptr) or already created but empty (!= nullptr but no windows)
-            if (workspacePtr == nullptr || g_pCompositor->getWindowsOnWorkspace(workspacePtr->m_iID) == 0) {
+            if (workspacePtr == nullptr || workspacePtr->getWindows() == 0) {
                 return workspaceName;
             }
         }

